package.json

```diff
@@ -19,10 +19,11 @@
     "bootstrap": "lerna bootstrap"
   },
   "dependencies": {
+    "babel-plugin-module-resolver": "^4.0.0",
+    "gatsby-plugin-why-did-you-render": "^2.0.0",
```

Comment:
Sanity check: does this plugin do anything in the build right now? If not, I'm fine with keeping it in as long as it doesn't hurt performance

Reply:
I ran `yarn remove`? I guess idk how to actually remove packages